apitools/base/protorpclite/messages.py

```diff
@@ -250,8 +251,7 @@ class MyMessage(Message):
         outer_definition_name = cls.outer_definition_name()
         if outer_definition_name is None:
             return six.text_type(cls.__name__)
-        else:
-            return u'%s.%s' % (outer_definition_name, cls.__name__)
+        return u'%s.%s' % (outer_definition_name, cls.__name__)
```

Comment:
Yup, agree that early return is more readable/maintainable. This particular instance seemed neither here nor there, but it is the base case of the pattern :)